cdba.h

```diff
@@ -10,7 +10,7 @@
 
 struct msg {
 	uint8_t type;
-	uint16_t len;
+	uint32_t len;
```

Comment:
this breaks the protocol

Reply:
+1. This needs to be fixed. Please keep uint16_t.